Fact sheet

strbean opened the pull request:
Adds Facts and Stats page with lots of static content and live totals for variants.

Also remove old links from navbar, remove CIMB logo from front page, and add ETH Zurich and UCSC Genomics Institute logos.

zfisch commented on website/js/FactSheet.js:
```diff
@@ -0,0 +1,152 @@
+    },
+    componentWillMount: function() {
+        backend.variantCounts().subscribe(
+            resp => this.setState(this.setState(resp)),
```
never seen this before (setState inside a setState call), is this intentional and necessary?

zfisch commented on website/js/FactSheet.js:
```diff
@@ -0,0 +1,152 @@
+            () => this.setState({error: 'Problem connecting to server'}));
+    },
+    render: function () {
+        //var {data, page, totalPages, error} = this.state;
```
pls remove comment if not needed

strbean commented:
I'll fix all those issues right away! No idea how I managed to put that
nested setState, I'm surprised it didn't blow up when I tested it ><

On Apr 24, 2017 9:51 AM, "Zack Fischmann" <notifications@github.com> wrote:

> *@zfisch* commented on this pull request.
>
> Looks good!
>
> I left a couple comments and am also curious if all images are needed --
> we should exclude any unused images from the codebase.
> ------------------------------
>
> In website/js/FactSheet.js
> <https://github.com/BRCAChallenge/brca-exchange/pull/454#discussion_r112993099>
> :
>
> > +'use strict';
> +
> +var React = require('react');
> +var PureRenderMixin = require('./PureRenderMixin'); // deep-equals version of PRM
> +var {Grid, Col, Row, Alert} = require('react-bootstrap');
> +var backend = require('backend');
> +var {Link} = require('react-router');
> +
> +var FactSheet = React.createClass({
> +    mixins: [PureRenderMixin],
> +    getInitialState: function() {
> +        return {};
> +    },
> +    componentWillMount: function() {
> +        backend.variantCounts().subscribe(
> +            resp => this.setState(this.setState(resp)),
>
> never seen this before (setState inside a setState call), is this
> intentional and necessary?
> ------------------------------
>
> In website/js/FactSheet.js
> <https://github.com/BRCAChallenge/brca-exchange/pull/454#discussion_r112993178>
> :
>
> > +var {Grid, Col, Row, Alert} = require('react-bootstrap');
> +var backend = require('backend');
> +var {Link} = require('react-router');
> +
> +var FactSheet = React.createClass({
> +    mixins: [PureRenderMixin],
> +    getInitialState: function() {
> +        return {};
> +    },
> +    componentWillMount: function() {
> +        backend.variantCounts().subscribe(
> +            resp => this.setState(this.setState(resp)),
> +            () => this.setState({error: 'Problem connecting to server'}));
> +    },
> +    render: function () {
> +        //var {data, page, totalPages, error} = this.state;
>
> pls remove comment if not needed
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/BRCAChallenge/brca-exchange/pull/454#pullrequestreview-34348760>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFULxvcxUBjHJfo1L7R187vaj2jA4mIaks5rzNMGgaJpZM4ND9ge>
> .
>

strbean commented:
Alright, I fixed those issues. I removed images that were made redundant by logo changes, but I didn't do a thorough search to see if any of the older images were already unused.
